### DISCUSSION TITLE
.5 pBTC is not distinguishible after sending through the wormhole eth to salano

### DISCUSSION THREAD

**Discussion by apollo711** [wormhole-foundation/wormhole#426] [Developer Q&A] [answered]
Started out to farm pBTC to renBTC on Saber Protocol on Solana Blockchain.  No problems with renBTC but had a lot of trouble getting the pBTC wrapped then into a wallet in order to stake it   The token is no longer distinguishable as pBTC.  It's now been several days and I'm trying to remember all of the steps I went through.  I could not get it into the pool 1st I tried Phantom wallet then Sollet neither worked.
Before any of this I tried going to Saber to get support but the dev made some comment about putting me through Defi 101 and it was off to the races...Anyway, I need help.  This is .5 of BTC and I'm becoming frustrated.  I previously tried reaching out to pBTC but there are db scammers everywhere in the space.  Anyway, I think the guy was a scammer, had to say bye to him.
Was thinking to reach out to a Solansa dev but want to take it through the best possible options.  So not doing anything until trying to speak to wormhole and p-token again!  My first experience with p-token was so weird, I think this guy was a bad actor.  He told me I was talking to the wrong firm and that there was another with the same logo…this is when I panicked and got the heck out of the DM,  
I would like to have a dev from p-token and Wormhole look at Block Explorer.  Not sure what else to do?  But as of now I have .5 of a token that was wrapped pBTC.  But no longer even resembles BTC or pBTC but bear in mind I’m not a dev and really don’t know what I’m talking about.
When I was unable to get the token in the renBTC to pBTC pool.  I moved it from Phantom wallet to Sollet wallet not to mention sometime after wrapping pBTC I sent it through the wormhole.  Okay hope this makes some sense.
I have attached a video clip so that you can see what I’m seeing.  Also, this is the address of the Phantom wallet where the .5 pBTC/BTC currently resides.  Thank you for your help.  

https://user-images.githubusercontent.com/47486784/133005551-c806cce2-fc1b-4210-a276-8c06c8d8fdd8.mp4

PeiHH3Va7Tn7tNWCQrdAng64rc9xANiuTB6NtAJFbw3

**Comment by leoluk** [comment, marked as answer]
Looks like this is the mint in question: https://explorer.solana.com/address/3oD7PhZstGGxcpLfxsZW8iGby8DQjSB3SJy74NPVHhgT

You should be able to unwrap them by sending them back to Eth using https://www.wormholebridge.com

**Reply by apollo711** [reply]
okay...that's kinda what I was thinking but I'm concerned that it may be the combination of first wrapping the pBTC and then sending it through the wormhole.  What if these protocols are not playing nice with each other?  I guess there is only one way of finding out.  I'll be trying in about an hour and will let you know.  I want to take another look at the block explorer before I try.

**Reply by apollo711** [reply]
Okay 99% sure I used v1 of the wormhole, can I confirm this in block explorer?

**Reply by leoluk** [reply]
Yes, it's Wormhole v1. The UI moved to https://v1.wormholebridge.com

**Reply by apollo711** [reply]
Yep, got it...now I have a fundamental problem.  I cannot select the token cuss it's unidentifiable.  As I'm sure you are aware the wormhole wants to pair a specific token.  I cannot select the token.  The wormhole is not recognizing it.  Please review vid clip for details.  I had previously tried to reach out to p Parrot that Saber has the partnership with but apparently there is another Defi firm with the same name?  Yea, I don't know.  Very confusing.  Not sure who to reach out to other than Saber and they have been 0 help.  Check out the video clip.  File is too big to upload so I have provided the file via OneDrive link

https://1drv.ms/v/s!Ar8pGyjRm3fhgrlVMlJ6JrtQhfkqNg?e=3f1nsR

**Reply by leoluk** [reply]
We do not maintain the Wormhole v1 UI, raise an issue here: https://github.com/solana-labs/oyster/issues

**Reply by leoluk** [reply]
https://github.com/solana-labs/oyster/issues/370

**Comment by apollo711** [comment]
Hello Leo I have made many attempts at getting this coin sent back through the v1 wormhole. I have tried employing the help of SS in the wormhole telegram but there are many scammers in the cesspool that is telegram and having had many try to scam me I'm not confident in the security and reliability of the platform. Besides the fundamental issues with the v1 platform and it's instability I have had much trouble trying to make it work. I think I can say with confidence that what it boils down to is the I can not select the token because the token picker is not configured to pick a coin that is no longer identifiable. v1 is configured for a limited number of tokens. I cannot select a coin/token whatever it is, that the wormhole is not configured to select. I have spent countless hours trying to find a way to make this work. I am not a developer and do not have the necessary technical expertise to develop the code that may or may not allow this to work. Even if the token picker is somehow configured to select the coin that what was once pBTC, there is no guarantee that sending it back through the portal will fix the issue. Although there have been great progress in the development of the wormhole protocol I have suspected sense early on that the team was under a lot of pressure to role out the protocol as the Solana blockchain was greatly dependent upon the wormhole to bring non-native assets onto the Solana ecosystem and thereby adding significant total value locked (TVL). But this rush to market has consequences and I have long been a victim of those consequences. I have tried to do whatever I can to make progress and to come up with a solution. Unfortunately I do not possess the necessary technical skills to bring this issue to resolution. It's time for Solana, Wormhole, and Sabor to be held to accountability and to do something more to resolve the issue. Until you have the the undistinguishable token in your custody it will be difficult to determine resolution. I propose we swap the .5 undistinguishable pBTC for an equal amount of bitcoin. This will allow you to test and evaluate and decide if making the necessary v1 configuration changes is will allow the coin to be restored at the same time making me whole and resolving my issue. Please advise how you would like to proceed.

**Comment by ckeun** [comment]
hey @apollo711, can you try this link?
https://v1.wormholebridge.com/#/move?from=SOL&token=PBTC
& see if it picks up your wrapped pBTC?

**Reply by apollo711** [reply]
ckeun, please read the complete string in detail in order to have a complete understanding of the issue.     The link you have provided is for the v1 wormhole.  Be advised that I have spent many hours trying to have the token picked and sent back through the wormhole.  v1 will not pick the token because it does not recognize it as pBTC.  The token is undistinguishable and is no longer recognizable by the protocol.  The v1 is codded and configured to select the tokens that it was originally developed for.  My token is no longer recognizable.  This issue has been going on for almost a year.  What happened to leoluk?  It has had open issues on two different platforms and has been scuffled around.  The original issues were closed without resolution.  Not sure why??  With all due respect, I am becoming frustrated with the lack of any significant progress.  This is the result when protocols are rushed to market without prober testing and vetting.  I'm not even sure why this is in the discussions tab.  This is an open Issue, in the issues log.  As I have previously advised, I am NOT a developer.  I very seldom have to interreact in the github repository.  I just happened to come across this discussion when I was checking to see if there had been any new developer review.  In my humble opinion, it is time to escalate this issue to a higher authority.  This is .5 of a bitcoin.  I would like to have my bitcoin back.  I really do not care if it is native BTC or if it is wrapped pBTC.  I just would like to have it back.  I regret ever bringing my liquidity to the Saber protocol.  Saber has been 0 help.  I originally expected to realize gains in exchange for providing liquidity.  Never happened with this half of my bitcoin.  Again, if you are a developer I suggest you go back and read the previous communications.  And please advise me if you are able to escalate this issue to a higher authority?  I believe it is time for me to turnover the token to the development team.  I'm happy to swap out the token for .5 of a wrapped BTC of any variety.  That way the team can investigate the issue and decide the resolution.  See attached screenshot of the token as it is in the Phantom wallet.  I await your reply.
3oD7PhZstGGxcpLfxsZW8iGby8DQjSB3SJy74NPVHhgT
[v1 pBTC 010.pdf](https://github.com/certusone/wormhole/files/8793981/v1.pBTC.010.pdf)
[v1 pBTC 011.pdf](https://github.com/certusone/wormhole/files/8793982/v1.pBTC.011.pdf)
